src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -44,6 +44,15 @@
                         {% set items = items|merge([{ 'value': content.name }]) %}
                         {% include '@ezdesign/parts/breadcrumbs.html.twig' with { items: items } %}
 
+                        {% if location.contentInfo.isHidden %}
+                            <div class="ez-alert--info">
+                                <svg class="ez-icon ez-alert--icon">
```

Comment:
`ez-alert--icon` not follow the BEM convention.
I think this should be `ez-icon--hide`